script/openqa-enqueue-needle-versions-cleanup

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -e
+exec "$(dirname "$0")"/openqa eval -m production -V 'app->gru->enqueue(remove_needle_versions => [], {priority => 5, ttl => 172800, limit => 1})' "$@"
```

Comment:
We also need systemd service and timer units and `openqa-webui.service` needs to depend on the timer unit.